docker/buildDocker.sh

```diff
@@ -138,34 +144,40 @@ buildDocker()
 	if [ -n "$bootDir" ]; then
 		commandString="$commandString -J $bootDir"
 	fi
-	if [[ "$openJ9" = true ]]; then
+	if [[ "$buildVariant" == "openj9" ]]; then
 		commandString="$commandString --build-variant openj9"
 	fi
 	for jdk in $jdkVersion
 	do
 		echo "$commandString $jdk being executed"
-		cd $WORKSPACE/DockerBuildFolder/openjdk-build && $commandString $jdk
+		cd $WORKSPACE/DockerBuildFolder/$jdk-$buildVariant && $commandString $jdk
 	done
 }
 
+setupFolders()
```

Comment:
This whole section can just be replaced by
`mkdir -p $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant`
which will create the full path and also not complain if the directory is already there

Reply:
Sorted :)